### DISCUSSION TITLE
Ubuntu update from 22.04 to 24.04 on kukui / Juniper

### DISCUSSION THREAD

**Discussion by gallee** [velvet-os/imagebuilder#257] [Q&A]
Hello,

on kukui / juniper, I tried to upgrade my ubuntu 22.04  with Canonical buit in mecanism (after doing a backup !).

The boot with 6.6.9-stb-mt8+` is OK but wifi is not working. Looking at kernel log, it seems related to 
```
[ 22.486192] remoteproc remoteproc0: Direct firmware load for mediatek/mt8183/scp.img failed with error -2
```

see dmesg https://pastebin.com/s1R9jLkH

Is there something obvious / easy or shall I restore my 22.04 ?

Thanks !

**Comment by thenameisluk** [comment]
you might be able to fix it by restoring firmware files if these being missing is the issue

look below

https://github.com/hexdump0815/imagebuilder/issues/199#issuecomment-1950300170

**Comment by gallee** [comment]
@thenameisluk  thanks that did the trick !
